polyglot/piranha/src/models/rule.rs

```diff
@@ -219,7 +232,7 @@ impl Rule {
     };
     for rule in rules {
       for ancestor in &context() {
-        if let Some(edit) = rule.get_edit(&source_code_unit.clone(), rules_store, *ancestor, false)
+        if let Some(edit) = rule.get_edit(&source_code_unit.clone(), rules_store, *ancestor, false, &None)
```

Comment:
Ensure that the next reported match occurs before (line-column number wise) the previously reported match.